src/App.jsx

```diff
@@ -1,4 +1,7 @@
 import React from 'react';
+
+import { Person } from './components/Person/Person';
+
```

Comment:
remove unnecessary lines
```suggestion
import { Person } from './components/Person/Person';
```